stock_location_orderpoint/models/stock_move.py

```diff
@@ -94,3 +85,16 @@ def _action_done(self, *args, **kwargs):
             ],
         )
         return moves
+
+    def _register_hook(self):
+        super()._register_hook()
+
+        def _patched_action_assign(self, *args, **kwargs):
```

Comment:
Personally I do not like at all such things.

Could you explain why you need it ?

Reply:
This ensures that the method _prepare_location_orderpoint_replenishment gets called at the end of the chain, 
also if an other module overwrites action_assign like https://github.com/OCA/wms/tree/14.0/stock_move_source_relocate.
Also if it gets overwritten like this, then we don't need the glue module stock_location_orderpoint_source_relocate anymore.

Reply:
I'd prefer glue module.

Reply:
I have removed the register_hook